rules/do-you-calculate-payroll-tax-correctly/rule.md

```diff
@@ -2,7 +2,7 @@
 seoDescription: Calculate payroll tax correctly by understanding Australian government rules and state-specific exemptions, avoiding common errors and staying up to date with changes.
 type: rule
 archivedreason:
-title: Do you calculate Payroll tax (Australia) correctly?
+title: Paryoll - Do you calculate Payroll tax correctly? (Aussie Only)
```

Comment:
Hi @JimmyChenSSW 

On all "Payroll" you made a typo

- [ ] 1. Please fix

Reply:
Done - That was a stupid error play around with my Copy+Paste.
Thank you Tiago!